book/src/external-attacher.md

```diff
@@ -8,7 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-attacher v3.0.2](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.0.2) | [release-3.0](https://github.com/kubernetes-csi/external-attacher/tree/release-3.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-attacher:v3.0.2 | v1.17 | - | v1.17
```

Comment:
Why is 3.0 removed?

Reply:
I replaced it because the min/max/recommended kubernetes versions are the same, so it is safe for vendors to skip over 3.0

Reply:
I agree that we shouldn't list versions that are definitely replaced by something else.

OTOH, we do list also some historic versions that are no longer supported. Would it make sense to document explicitly which of the listed versions are still supported?

Reply:
Yes, I could move them to a new table or add a new column. Or add a note that we officially only support the last X versions

Reply:
I prefer a column. Less churn that maintaining a separate table and more obvious than letting everyone figure what the last X versions are.

Reply:
I've been thinking about this a little more, and I think it would be simpler to have a top level support policy page instead of having to update these all these tables every release. I prefer to work on that separately.

Reply:
Okay, let's address that separately. Once there is a draft we'll see how easy it is to map that to specific sidecars.